gamefixes-steam/1144400.py

```diff
@@ -0,0 +1,13 @@
+""" Game fix for Senren Banka
+"""
+#pylint: disable=C0103
+
+from protonfixes import util
+
+def main():
+    """
+    """
```

Comment:
It's the empty docstring which is unnecessary. See the linter errors for more details

Reply:
and does this mean i have to submit another pull request with those gone in the script